hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/test/java/org/apache/hadoop/yarn/server/federation/store/impl/TestSQLFederationStateStore.java

```diff
@@ -104,7 +105,7 @@ protected FederationStateStore createStateStore() {
     conf.set(YarnConfiguration.FEDERATION_STATESTORE_SQL_PASSWORD,
         DATABASE_PASSWORD);
     conf.set(YarnConfiguration.FEDERATION_STATESTORE_SQL_URL,
-        DATABASE_URL + System.currentTimeMillis());
+        DATABASE_URL + System.currentTimeMillis() + MYSQL_COMPATIBILITY);
```

Comment:
Is this ignored by others?

Reply:
INSERT IGNORE INTO is Mysql syntax, not applicable to other databases.

Reply:
I meant the MYSQL_COMPATIBILITY flag.
This test is run for other SQL implementations right? How do they handle that?

Reply:
I will refactor this part of the code to be compatible with other SQL implementations.